modules/openapi-generator/src/main/java/org/openapitools/codegen/languages/PythonClientCodegen.java

```diff
@@ -61,6 +61,7 @@ public PythonClientCodegen() {
         // clear import mapping (from default generator) as python does not use it
         // at the moment
         importMapping.clear();
+        importMapping.put("BigDecimal", "Decimal from decimal");
```

Comment:
This change is unrelated and generates many new files.
If you need this feature please open it in a separate PR.
Python assumes that non integer numbers are floats.

Can you remove this update from here?

Reply:
Done

Reply:
Can you resolve this conversation?